mlflow/genai/experimental/databricks_trace_archival.py

```diff
@@ -54,6 +57,17 @@ def _validate_schema_versions(spans_version: str, events_version: str) -> None:
     )
 
 
+def _get_spark_session():
+    try:
+        from pyspark.sql import SparkSession
+
+        return SparkSession.builder.getOrCreate()
+    except Exception:
```

Comment:
Can we check if databricks-connect is installed? It'll be confusing if it's not remote spark session issue but some other issue, can we try catch all exception here